versions.props

```diff
@@ -1,4 +1,4 @@
-com.diffplug.spotless:spotless-plugin-gradle = 4.4.0
+com.diffplug.spotless:spotless-plugin-gradle = 5.7.0
```

Comment:
confirmed this is in managed-artifacts.txt :)

Reply:
Yep, this excavator also tries to upgrade it but the tests fail because of the breaking changes: https://github.com/palantir/gradle-baseline/pull/1536/commits/286964eaec3f6af3c4b9071abee6e6e37b1ea6ac#diff-c2c619cc6bd6031ee10ada015c7ba9b841732e59d0a05d2a7408ebb0c02501e9L1-R1